packages/palette-docs/content/docs/elements/dialogs/Modal.mdx

```diff
@@ -24,6 +33,7 @@ name: Modal
               width="100%"
               onClick={() =>
                 setState({
+                  contentKey: "456",
```

Comment:
Can one more example be added that shows the content in the modal updating immediately, without fade in / fade out?

Reply:
Just added